developers/weaviate/more-resources/migration/weaviate-1-30.md

```diff
@@ -0,0 +1,325 @@
+---
+title: BlockMax WAND migration guide
+sidebar_label: 1.30 (BlockMax WAND migration)
+sidebar_position: 9
+image: og/docs/more-resources.jpg
+---
+
+This comprehensive guide covers how to migrate existing data to use the BlockMax WAND algorithm for the inverted index in Weaviate `v1.30` and above.
+
+**BlockMax WAND** offers improved search performance for BM25 and hybrid searches. While new collections created in Weaviate `v1.30` or newer will automatically use this optimized format, existing collections created before `v1.30` require migration to take advantage of these improvements.
+
+To learn more about the technical details of this change, you can refer to [the BlockMax WAND blog post](https://weaviate.io/blog/blockmax-wand).
+
+## Prerequisites
+
+Before beginning the migration, ensure that:
+
+- You have a Weaviate version before `v1.30` that doesn't use the BlockMax WAND algorithm
+- You're on the latest version of Weaviate before migrating (currently `||site.weaviate_version||`)
+
+## Migration considerations
+
+Before starting the migration process, be aware of the following:
+
+- Migration is recommended primarily if you use BM25 or hybrid search and aren't satisfied with your current query times
+- Migration will increase server resource load, especially during the re-indexing stage
+- Migration can take hours for shards with millions of documents, depending on the number of searchable properties
+- If possible, schedule migration to avoid large data imports/updates/deletes, especially during the re-indexing stage
+- BlockMax WAND may return slightly different scores than WAND if you search on multiple properties or have a delete-heavy workflow, as it computes term statistics differently
+
+## Migration process
+
+The migration involves three main stages:
+
+1. **Re-indexing**: Converting the internal representation from `mapcollection` to the inverted `blockmax` format
```

Comment:
Hey! Wdyt about:

1. **Re-indexing**: Build a new `blockmax` format index based on the existing `mapcollection` index.
2. **Swapping**: Update Weaviate configurations to use the new format while maintaining the ability to roll back